src/app/personsok/Sokeresultat.tsx

```diff
@@ -22,6 +22,8 @@ function Sokeresultat(props: Props) {
     );
     if (trengerUtenlandskIDTittel) {
         tittelRekke.push('Utenlandsk ID');
+    } else {
+        tittelRekke.push('');
```

Comment:
Dette vil gjøre at det kommer en ekstra kolonne selvom vi ikke trenger det. 
Er det med vilje?

Reply:
Ja, det ble gjort for å få lik lengde på `tittelRekke`og ´tableEntries` siden jeg fikk en feilmelding i console om at lengden på tittelRekke og lengden på tableEntries var av forskjellig lengde når vi ikke hadde noen utenlandsk ID.

Reply:
```
    const tableEntries = props.response.map(linje => {
        const entries = [
            <IdentCelle ident={linje.ident} />,
            <NavnCelle navn={linje.navn} status={linje.status} />,
            <AdresseCelle response={linje} />,
            <BostedCelle brukerinfo={linje.brukerinfo} />,
        ];
        if(trengerUtenlandskIDTittel){
           entries.push(
               <UtenlandskIDCelle utenlandskID={linje.utenlandskID}/>
           )
        }
        return entries
    });
```